state/core/src/main/java/com/expediagroup/streamplatform/streamregistry/state/DefaultEntityView.java

```diff
@@ -47,10 +47,19 @@ public class DefaultEntityView implements EntityView {
     this(receiver, entities, new DefaultEntityViewUpdater(entities));
   }
 
+  DefaultEntityView(EventReceiver receiver, Map<Entity.Key<?>, StateValue> entities, Boolean entityStatusEnabled) {
+    this(receiver, entities, new DefaultEntityViewUpdater(entities, entityStatusEnabled));
+  }
+
+  public DefaultEntityView(EventReceiver receiver, Boolean entityStatusEnabled) {
+    this(receiver, new ConcurrentHashMap<>(), entityStatusEnabled);
+  }
+
   public DefaultEntityView(EventReceiver receiver) {
     this(receiver, new ConcurrentHashMap<>());
   }
 
+
   @Override
   public CompletableFuture<Void> load(@NonNull EntityViewListener listener) {
     val future = new CompletableFuture<Void>();
```

Comment:
Adding an optional `entityStatusEnabled` flag to all constructor patterns will can be used to disable entity status persistence.